frontend/src/_layouts/base.html

```diff
@@ -79,45 +79,24 @@
         >
-          <!--
-            Currently Parcel keeps html extension in the references.
-            https://github.com/parcel-bundler/parcel/discussions/6903
-            Possible solution: https://v2.parceljs.org/languages/html/#:~:text=This%20example%20uses%20posthtml-shorten%20to%20shorten%20URLs%20using%20a%20custom%20shortening%20function.
-          -->
-          <!-- Currently it is not possible to pass data from page to the layout: https://github.com/posthtml/posthtml-extend/issues/21 -->
-          <!-- Therefore override corresponding *-menu-item block in a page to make it styled active -->
-          <!-- In the future we can leverage PUG: https://stackoverflow.com/a/29089926/3082178 -->
-          <block name="audios-menu-item">
-            <a href="/">Audios</a>
-          </block>
-          <block name="memories-menu-item">
-            <a href="/memories">Memories</a>
-          </block>
-          <a href="https://www.audioseva.com/donate" target="_blank">Donate</a>
-          <a href="https://docs.google.com/forms/d/e/1FAIpQLSfjwP_n5uJPD-BxwE_Ta-tbaf2aWNWZqe-YOOoUzjcPw209Uw/viewform" target="_blank">Contact</a>
-          <block name="about-menu-item">
-            <a href="/about">About</a>
-          </block>
-          <block name="join-menu-item">
-            <a href="/join">Join</a>
-          </block>
-        </div>
+          {% assign navPages = collections.all | eleventyNavigation %}
+          {% for entry in navPages %}
+            <a href="{{ entry.url }}" {% if entry.url == page.url %} class="rounded-md bg-teal-800 text-white"{% endif %}  {% if entry.title == "donate" %} target="_blank" {% endif %}  {% if entry.title == "contact" %} target="_blank" {% endif %} > {{ entry.title }} </a>
+          {% endfor %}
+       </div>
       </div>
     </nav>
-    <block name="banner">
-      <div class="mx-auto flex w-full max-w-screen-xl flex-wrap items-center justify-center space-x-1 border-b px-4 py-2 text-sm sm:pl-36 md:pl-56 lg:pl-72">
-        <a class="inline-flex items-center rounded-md bg-red-400 px-2 py-1 text-white shadow-sm" href="/join">JOIN US</a>
-        <span class="py-1 text-center">for <strong>Daily Harikatha &amp; Short Videos</strong> in Hindi &amp; English </span>
-      </div>
-    </block>
+    <!-- banner -->
+    <!-- will use alpine x-show directive to hide the banner element later -->
+    <div class="mx-auto flex w-full max-w-screen-xl flex-wrap items-center justify-center space-x-1 border-b px-4 py-2 text-sm sm:pl-36 md:pl-56 lg:pl-72 {% if bannerShow == false %} hidden {% endif %}">
+      <a class="inline-flex items-center rounded-md bg-red-400 px-2 py-1 text-white shadow-sm" href="/join">JOIN US</a>
+      <span class="py-1 text-center">for <strong>Daily Harikatha &amp; Short Videos</strong> in Hindi &amp; English </span>
+    </div>
```

Comment:
Just don't render this block altogether if `bannerShow == false`. I mean, enclose it all in the `if` block.

Reply:
the banner `div` is enclosed inside `if` block, not rendering in join page.